### ISSUE TITLE
Inlay hints for function parameter names

### ISSUE THREAD

**Issue by ylazy** [BowlerHatLLC/vscode-as3mxml#709]
Hi!

It would be nice if we have something like [this](https://marketplace.visualstudio.com/items?itemName=liamhammett.inline-parameters) for ActionScript. Much more readable codes.

Please consider to support this feature.

Thank you!

**Comment by ylazy** [comment]
VSCode has this setting "editor.inlayHints.enabled":"on" enabled by default, but it seems to only work with typescript.

**Comment by joshtynjala** [comment]
> but it seems to only work with typescript

Yeah, to get this feature working in AS3, I would need to implement the [Inlay Hint Request](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_inlayHint) feature in the language server protocol.

**Comment by joshtynjala** [comment]
I have added inlay hints for parameter names in my latest commits. To enable them, `as3mxml.inlayHints.parameterNames.enabled` must be set to either `"all"` or `"literals"`

```
"editor.inlayHints.enabled":"on",
"as3mxml.inlayHints.parameterNames.enabled": "all"
```

I observed that `typescript.inlayHints.parameterNames.enabled` defaults to `"none"`, so the AS3 version uses the same default.

**Comment by pngdrift** [comment]
Inlay hints not displayed for the class constructors (e.g. `new SomeClass(...)`). Is that the way it should be?
